src/deploy/NVA_build/noobaa_init.sh

```diff
@@ -214,14 +214,14 @@ init_noobaa_agent() {
 
   cd /root/node_modules/noobaa-core/
   prepare_agent_conf
-  run_internal_process node ./src/agent/agent_cli
+  run_internal_process node --unhandled-rejections=warn ./src/agent/agent_cli
```

Comment:
@utkarsh-pro Did you consider setting it in the NODE_OPTIONS env instead of adding to all calls?
See https://nodejs.org/docs/latest-v16.x/api/cli.html#node_optionsoptions
But if we are going to clean up these warnings soon, as we should, then it doesn't matter anyway.

Reply:
@guymguym, actually I did open up an issue for cleaning this up here: https://github.com/noobaa/noobaa-core/issues/7083. But as I mentioned in the issue, I think we would need a tiny bit more help from the runtime as right now it doesn't event shows that the `uncaughtException` was rooted from `unhandledRejection` and has no traces of it in the stack trace either. That's why resorted to this.

I think I completely missed considering the `node options` though, thank you!